src/olympia/bandwagon/admin.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib import admin
+
+from .models import Collection, CollectionUser
+
+
+class ContributorInline(admin.TabularInline):
+    model = CollectionUser
+    raw_id_fields = ('user',)
+    fields = ('user',)
+    verbose_name_plural = 'Contributors'
```

Comment:
Let's localize all the things

Reply:
Sure.  Though a total of 3 people will see this in django admin on prod and they can all read English :)

Reply:
https://github.com/mozilla/addons-server/pull/6494/commits/3a06d116dbe80282f6ff2673678a0ad699f1baef